.pre-commit-config.yaml

```
@@ -0,0 +1,32 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: no-commit-to-branch
+        args: [-b, main, -b, develop]
+      - id: detect-private-key
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+      - id: check-merge-conflict
+
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 24.1.1
+    hooks:
+      - id: black
+        language_version: python3.12
+        exclude: migrations
+
```

```suggestion
        exclude: migrations
        args: [--line-length=79]

```
Если будем делать по PEP, то надо уменьшить длину строки для black

---

Или 88 оставим?

---

Надо обсудить со всеми. Мне кажется, удобнее 88. Вечно что-то не влезает, а потом непонятно, как это переносить.)))